lua/entities/gmod_wire_expression2/init.lua

```diff
@@ -682,7 +682,7 @@ local function enableEmergencyShutdown()
 					current_ram > halt_max_amount:GetInt() * 1000 then -- or if the current ram goes over a set limit
 
 					local e2s = ents.FindByClass("gmod_wire_expression2") -- find all E2s and halt them
```

Comment:
I can do that in a future commit.

Reply:
> could probably do a table of e2s here too like you did with the pods

I don think that is too necessary since it's not called frequently. Whereas the rt screens was calling ents.FindByClass every tick for every player on the server which caused very severe performance impact. The pods were a similar story, but not as bad.